admin/app/components/solidus_admin/ui/panel/component.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class SolidusAdmin::UI::Panel::Component < SolidusAdmin::BaseComponent
+  # @param title [String] the title of the panel
+  # @param title_hint [String] the title hint of the panel
+  # @param actions [String] the rendered html for the actions section
+  # @block content [String] the rendered html for the content section
+  def initialize(title: nil, title_hint: nil, actions: nil)
+    @title = title
+    @title_hint = title_hint
+    @actions = actions
```

Comment:
Recently I've discovered that ViewComponent allows us to add new `slots` to the blocks we open when creating a component.
What do you think on turning the `actions` section into a slot? (Just like the content) So the dev can have more freedom instead of just passing the HTML on the initializer.
https://viewcomponent.org/guide/slots.html

like 
```
has_one :actions
```

and

```
   <%= render current_component.new(title: "SEO", title_hint: "Search Engine Optimization") do |panel| %>
     <label>
       SKU:
       ABC-123
     </label>
     
     <% panel.with_actions do %>
       ...HTML
     <% end %>
   <% end %>
   ```

Reply:
Thanks great call! I'll embed this change in the products#show PR and update this later on 👍👍👍